src/hocon_schema_md.erl

```diff
@@ -129,6 +132,8 @@ fmt_mapping(Str) -> ["`", Str, "`"].
 
 fmt_type(T) -> hocon_md:code(do_type(T)).
 
+do_type(#{kind := <<Kind/binary>>} = Type) ->
```

Comment:
not quite following why is this needed.

Reply:
When one reads the schema from an existing JSON file, the fields and values are binaries, so this was blowing up.

It didn't blow up before the refactoring because the schema was processed in-memory, without being written first to disk.  So the key/values stayed as atoms.

(at least that's what I remember, it has been some weeks).